qa/integration-tests/src/test/java/io/camunda/zeebe/it/smoke/ContainerClusterSmokeIT.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.smoke;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.zeebe.containers.cluster.ZeebeCluster;
+import io.zeebe.containers.engine.ContainerEngine;
+import java.util.concurrent.TimeUnit;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+final class ContainerClusterSmokeIT {
+
+  private final ZeebeCluster cluster =
+      ZeebeCluster.builder()
+          .withBrokersCount(1)
+          .withGatewaysCount(1)
+          .withPartitionsCount(1)
+          .withEmbeddedGateway(false)
+          .withImage(ZeebeTestContainerDefaults.defaultTestImage())
+          .build();
+
+  @Container
+  private final ContainerEngine engine = ContainerEngine.builder().withCluster(cluster).build();
+
+  /** A smoke test which checks that a gateway of a cluster can be accessed. */
```

Comment:
:question: The other smoke tests check that we can deploy a process, start it, and complete it. Here we only check the topology. I imagine this is because of the overlap between both tests? Or is what makes just checking the nodes connect enough for a smoke test?

Reply:
Would it cover code paths using native libraries? Though if we do get the other smoke tests running on ARM64 as well, then again, with the overlap, it might be fine to leave it as is. If not, then we might want to make sure we do a little bit of processing to ensure we use at least RocksDB.

Reply:
went for adding a second smoke test that deploys a model and runs an instance 1350c248a0f72f7c5754bb43422a623c041c8cb6